src/embed/VbrickEmbedConfig.ts

```diff
@@ -22,12 +22,10 @@ export interface VbrickEmbedConfig extends VbrickSDKConfig {
 	 */
 	hideVideo?: boolean;
 
-
 	/**
-	 * Plays the video in a popup window. If false(default) the video plays directly in the embed element.
-	 * Only implemented for VOD Embeds
+	 * For video embeds. If the user needs to log in, show the login form in a popup window.
```

Comment:
This also applies for Single Sign On workflows, not just login form. Maybe "If a user needs to log in go through the login process in a popup window. This is the standard behavior for non-SDK Rev embeded videos"